lang/de/lang.php

```diff
@@ -7,7 +7,7 @@
 // Menüeintrag für Admin-Plugins
 $lang['menu'] = 'Wiederkehrende Überprüfungen verwalten';
 // custom language strings for the plugin
-$lang['maintained by'] = 'Wird überprüft von %s.';
+$lang['maintained by'] = 'Wird überprüft von: ';
```

Comment:
Why dropping the sprintf?

Reply:
I replaced string concatenation with sprintf again.